### PR TITLE
POL-181 - Error Azure Publicly Accessible SQL Managed Instance

### PR DESCRIPTION
**Pull request by praut1**
### Description

Modified existing policy to resolve an error regarding tags

### Issues Resolved

[List any existing issues this PR resolves]

### Contribution Check List

- [x] All tests pass.
- [x] New functionality includes testing.
- [x] New functionality has been documented in the README if applicable
- [x] New functionality has been documented in CHANGELOG.MD

Running Template link and screenshot

Azure Publicly Accessible Managed SQL Instance
https://governance.rightscale.com/org/1105/projects/60073/policy-incidents/5e30196d4ec7b300014e7559
image

### CODE REVIEW

File: security/azure/sql_publicly_accessible_managed_instance/README.md [hunk 1]
```diff
@@ -1,62 +1,50 @@
...
-- *Exclusion Tag Key* - Azure SQL Managed instance tag to ignore instance that are with public data endpoint enabled. Only supply the tag key. The policy assumes that the tag value is irrelevant.
-- *Azure AD Tenant ID* - the Azure AD Tenant ID used for the Azure API Authentication
-- *Azure Subscription ID* - the Azure Subscription ID used for the Azure API Authentication
+- Edit the file [Check_for_publicly_accessible_Azure_SQL_Managed_Instance](https://github.com/flexera/policy_templates/tree/master/security/azure/sql_publicly_accessible_managed_instance)
+- uncomment the line which conatins 'escalate $esc_delete_Managed_instances_approval' and save the changes.
+- upload the modified file and apply the policy.
+
+## Azure Required Permissions
+
+- Microsoft.Sql/managedInstances/*
```
Comment (cdwilhelm) [inline security/azure/sql_publicly_accessible_managed_instance/README.md]: Remove the Azure Required Permissions, but move the bullet under Cloud Configuration.  All of the credential and permissions should be under the Prerequisite

Comment (praut1) [reply security/azure/sql_publicly_accessible_managed_instance/README.md]: Removed the Azure Required Permissions and added the bullet under Prerequisite

File: security/azure/sql_publicly_accessible_managed_instance/check_for_publicly_accessible_azure_sql_managed_instance.pt [hunk 2]
```diff
@@ -98,11 +109,80 @@ datasource "ds_list_sql_managed_instances" do
...
+
+###############################################################################
+# Policy
+###############################################################################
+
+policy "pol_sql_managed_instances" do
+  validate_each $ds_sql_managed_instances do
+    summary_template "{{ rs_project_name }} (Account ID: {{ rs_project_id }}): {{ len data }} Azure SQL managed instances with public endpoint enabled"
+    detail_template <<-EOS
+# Azure SQL managed Instance Details
```
Comment (cdwilhelm) [inline security/azure/sql_publicly_accessible_managed_instance/check_for_publicly_accessible_azure_sql_managed_instance.pt]: Add sorting by subscription name and region.

Comment (praut1) [reply security/azure/sql_publicly_accessible_managed_instance/check_for_publicly_accessible_azure_sql_managed_instance.pt]: Sorting logic has been added

**Pull request merged**